File: beginner/javascript/1183.js
let input = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n'); 

let operation = input.shift();
let elementos_acima_diagonal = 0;

let matrice = new Array();

for(let i = 0; i < 12; i++){ 
    matrice.push(new Array());
    for(let j = 0; j < 12; j++){
        matrice[i][j] = parseFloat(input.shift());
    }
}
    
const operation_in_line = (mat, op) => {
    let sum = 0;
    for (i = 0; i < 12; i++) {
        for (j = i + 1; j < 12; j++) { 
            sum += mat[i][j];
            elementos_acima_diagonal++;
        }
    }
    if(op === 'S') return sum;
    else return sum / elementos_acima_diagonal;
};

console.log(operation_in_line(matrice, operation).toFixed(1));

File: beginner/python/1183.py
operation = input()

matrice = [[0 for j in range(12)] for i in range(12)]

for i in range(12):
    for j in range(12):
        matrice[i][j] = float(input())

def operation_in_line(mat, op):
    sum = 0
    cont = 0
    for i in range(0,12):
        for j in range(i+1,12):
            sum += mat[i][j]
            cont+=1
            
    if op == 'S':
        return sum
    else:
        return sum / cont


print("{:.1f}".format(operation_in_line(matrice, operation)))
